### ISSUE TITLE
linux cpu 100%

### ISSUE THREAD

**Issue by calvin2021y** [floooh/cimgui-sokol-starterkit#20]
I test this on windows, mac, work well.

but on linux(I run a xrdp remote env), the cpu is up to 560%.

**Comment by floooh** [comment]
This is weird because sokol_gfx.h is strictly single-threaded. Maybe vsync throttling doesn't work on xrdp? But that would at best saturate a single CPU core.

**Comment by floooh** [comment]
Yeah, might be an issue with xrdp: https://github.com/neutrinolabs/xrdp/issues/362
